.github/actions/step_create_python_environment/action.yml

```diff
@@ -0,0 +1,21 @@
+name: step_create_conda_environment
+inputs:
+  pipFilePath:
+    required: false
```

Comment:
Please also refine `build_doc.yml`:
![image](https://github.com/microsoft/promptflow/assets/2572521/0b18bad6-41be-4d4f-8c10-fe739aad246d)

and `publish_doc.yml`:
![image](https://github.com/microsoft/promptflow/assets/2572521/7e86a62c-be2b-40db-9e27-11c9468fb306)

Reply:
Will this refine conflict with the `ROBOCOPY` thing we discussed before?

Reply:
No, please informed that robocopy need special steps and special windows machine to get it work.